Documentation/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/python:3.7.9-alpine3.11
+FROM docker.io/library/python:3.7.10-alpine3.12@sha256:ffe14871002b68eb7ca9583d7b0bf991d9066f7d9f9cb713c6510e966750ef19
```

Comment:
I know that alpine 3.13 is having some issues as I got one big incident related to dns recently, but didn't know 3.11 is also problematic. Can you share some details if I might ? Thanks.

Reply:
Simply deploy this snippet and you will see that alpine 11 and 13 will fail:
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    k8s-app: alpine-checker
  name: alpine-checker
  namespace: default
spec:
  replicas: 1
  selector:
    matchLabels:
      k8s-app: alpine-checker
  template:
    metadata:
      labels:
        k8s-app: alpine-checker
      name: alpine-checker
    spec:
      containers:
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.10
        imagePullPolicy: IfNotPresent
        name: alpine-3-10
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.11
        imagePullPolicy: IfNotPresent
        name: alpine-3-11
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.12
        imagePullPolicy: IfNotPresent
        name: alpine-3-12
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.13
        imagePullPolicy: IfNotPresent
        name: alpine-3-13
      dnsPolicy: ClusterFirst
      restartPolicy: Always
```

Reply:
thanks :100: , the above is nice :100: